setup.py

```diff
@@ -3,9 +3,9 @@
     name='consensus',
     version='1.0',
     packages=find_packages(),
-    install_requires=['molgenis-py-client>=2.1.0', 'progressbar2==3.39.3', 'termcolor==1.1.0', 'yaspin==0.14.3', 'pandas'],
+    install_requires=['progressbar2==3.39.3', 'termcolor==1.1.0', 'yaspin==0.14.3', 'pandas'],
     author='Mariska Slofstra',
```

Comment:
I think this needs to be replaced with something more generic. Based on the [molgenis repo](https://github.com/molgenis/molgenis/blob/master/pom.xml#L34) and [vip report API](https://github.com/molgenis/vip-report-api/blob/master/.travis.yml#L6) I'd suggest something like this, though if you have a better suggestion that's totally fine as well.
```
author='MOLGENIS development team',
author_email='molgenis@gmail.com',
```

See also: